README.md

```diff
@@ -21,18 +21,41 @@ We're working on [tutorials](https://napari.org/tutorials/), but you can also qu
 
 ## installation
 
-**napari** can be installed on most macOS, Linux, and Windows systems with Python 3.6, 3.7 and 3.8 by calling
+### from pip, with "batteries included"
+
+napari can be installed on most macOS, Linux, and Windows systems with
+Python 3.6, 3.7 and 3.8 using pip:
 
 ```sh
-$ pip install napari
+pip install napari[all]
 ```
 
+napari needs a library called [Qt](https://www.qt.io/) to run its user interface
```

Comment:
I think this is important discussion. I am also wondering if it is too detailed for the readme. In my mind, the install section readme should provide the minimal information to get going, targeted more at people who just want to use napari and less at "power users" and developers. My concern is that for a lot of users, I think the discussion about multiple qt libraries and the possibility to install without a qt library would raise more questions than it would answer.

Do you think it could make sense to just recommend `pip install napari[all]` and then link to the extended discussion somewhere else in the docs?

Also, have we gotten a lot of issues with people installing from pip and then having qt conflicts with other packages they have installed? Is it worth suggesting to install in a fresh environment?

Reply:
I updated the readme to move it lower down & put it in an "inset" block ... and added a blurb about environments.  look better?
https://github.com/napari/napari/blob/9557bc3b44eba5fd19898a0963463df87b075c85/README.md

Reply:
Looks good to me. Thanks, @tlambert03 !

Reply:
these changes look good to me @tlambert03!